### PR TITLE
Display total Org users and total Org plans statistics on the Usage s…

### PR DESCRIPTION
**Pull request by johnpinto1**
Fixes # 1760.

Changes proposed in this PR:
- Added functionality to display the Org User and plans totals.

### CODE REVIEW

File: app/views/usage/index.html.erb [hunk 1]
```diff
@@ -1,8 +1,24 @@
 <% title _('Usage statistics') %>
 
 <div class="row">
-  <div class="col-md-12">
+  <div class="col-md-12" >
     <h1>Usage statistics</h1>
+  </div>
+</div>
+<div class="row" style="margin: 15px 0;">
```
Comment (Bodacious) [inline app/views/usage/index.html.erb]: Can we avoid adding styles in the HTML. 

Please add CSS classes or use existing CSS for these.

**Comment by johnpinto1** [comment]
Updated code and moved from inline-styling to using css.

File: app/controllers/usage_controller.rb [hunk 2]
```diff
@@ -8,7 +8,7 @@ def index
           (current_user.can_org_admin? || current_user.can_super_admin?)
       raise Pundit::NotAuthorizedError
     end
-    render("index", locals: { orgs: Org.all })
+    render('index', locals: { orgs: Org.all, total_org_users: current_user.org.users.size, total_org_plans: current_user.org.plans.size })
```
Comment (briri) [inline app/controllers/usage_controller.rb]: Rubocop prefers double quotes

**Comment by johnpinto1** [comment]
@briri Updated code as suggested and forced pushed. So previous commit no longer present.

**Comment by briri** [comment]
looks good @johnpinto1. Thanks!

**Pull request merged**